### ISSUE TITLE
Can no longer access my wallet

### ISSUE THREAD

**Issue by Raffa19** [Multibit-Legacy/multibit-hd#987]
I've got a transaction which seems to be stuck (36 hrs so far), and today I have tried to open my wallet and the password isn't working.  Just tells me that it has failed to open the wallet.  I've no idea if the 2 things are connected in any way, but I more worried that I can't access my wallet than the transaction being stuck which should resolve itself eventually.  I do want to get at my bitcoins though!  I've tried going to 'forgotten my password' and using my words, although I am 100% confident that I have it right, and it has been working fine yesterday and earlier today.  After putting in my words, I just get a message saying 'working...', and that's all.  Really could do with some help - I'm having a nightmare!!

**Comment by Raffa19** [comment]
OK - Have done a fair bit of looking through other people's problems.  It seems the 2 issues are probably linked ... the problem isn't to do with the encryption, it's something to do with the content of the wallet itself.  I think I have got access back to the wallet by restoring to a version earlier than when I set up the transaction that's stuck.  I now have access to the wallet (I think), but am just waiting for it to synchronise from 40%.  Will close - next step is trying to get the transaction to go through somehow.
